protocol/v2/ssv/validator/opts.go

```diff
@@ -23,7 +23,7 @@ type Options struct {
 	BeaconNetwork     beacon.BeaconNetwork
 	Storage           *storage.QBFTStores
 	SSVShare          *types.SSVShare
-	Operator          *spectypes.Operator
```

Comment:
I would suggest the same rename here (`Operator` to `CommitteeMember`). If there's already the intention to do this in another PR, it's fine :)